rts/Lua/LuaSyncedRead.cpp

```diff
@@ -7027,17 +7040,37 @@ int LuaSyncedRead::GetFeatureResurrect(lua_State* L)
 /***
  *
  * @function Spring.GetFeatureLastAttackedPiece
+ * Returns nil if no feature found with ID.
  * @param featureID integer
+ * @return string|""|nil Last hit piece name
+ * @return integer frame it was last hit on
  */
 int LuaSyncedRead::GetFeatureLastAttackedPiece(lua_State* L)
 {
 	return (GetSolidObjectLastHitPiece(L, ParseFeature(L, __func__, 1)));
 }
 
+/*
```

Comment:
https://github.com/beyond-all-reason/spring/actions/runs/14583634856/job/40905494095?pr=2217

This class isn't being generated because its comment block doesn't start with `/***`.